### ISSUE TITLE
Use Legacy Swift Language Version (SWIFT_VERSION)

### ISSUE THREAD

**Issue by emcniece** [driftyco/ionic#8374]
#### Short description of the problem:

IOS builds are failing with the recent update to XCode 8 - "(SWIFT_VERSION) is required to be configured correctly..."

#### What behavior are you expecting?

Successful compilations. Setting the `SWIFT_VERSION=2.3` variable in the environment doesn't affect the build process.

**Steps to reproduce:**
1. Upgrade to XCode 8
2. Include a Cordova module that uses Swift ([Is-Debug](http://ionicframework.com/docs/v2/native/isdebug/) for example)
3. `ionic build ios`

```sh
Running 'build:before' npm script before build
> ionic-hello-world@ build /Users/emcniece/Code/ionic/app
> ionic-app-scripts build --dev
[15:59:52]  ionic-app-scripts 0.0.23
[15:59:52]  build dev started ...
[15:59:52]  clean started ...
[15:59:52]  clean finished in 10 ms
[15:59:52]  copy started ...
[15:59:52]  typescript compiler started ...
[15:59:52]  lint started ...
[15:59:52]  copy finished in 54 ms
[15:59:52]  tslint: /Users/emcniece/Code/ionic/app/src/providers/diagnostic/diagnostic.ts[4, 10]: Unused import: 'Diagnostic'


[15:59:52]  lint finished in 601 ms
[15:59:54]  typescript compiler finished in 2.30 s
[15:59:54]  bundle dev started ...
[16:00:00]  bundle dev finished in 6.41 s
[16:00:00]  sass started ...
[16:00:02]  sass finished in 1.23 s
[16:00:02]  build dev finished in 9.95 s
Running command: /Users/emcniece/Code/ionic/app/hooks/after_prepare/010_add_platform_class.js /Users/emcniece/Code/ionic/app


add to body class: platform-ios

Building project: /Users/emcniece/Code/ionic/app/platforms/ios/MyApp.xcodeproj

  Configuration: Debug
  Platform: emulator

Build settings from command line:

    CONFIGURATION_BUILD_DIR = /Users/emcniece/Code/ionic/app/platforms/ios/build/emulator
    SDKROOT = iphonesimulator10.0
    SHARED_PRECOMPS_DIR = /Users/emcniece/Code/ionic/app/platforms/ios/build/sharedpch

Build settings from configuration file '/Users/emcniece/Code/ionic/app/platforms/ios/cordova/build-debug.xcconfig':

    CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES
    CODE_SIGN_IDENTITY = iPhone Developer
    ENABLE_BITCODE = NO
    GCC_PREPROCESSOR_DEFINITIONS = DEBUG=1
    HEADER_SEARCH_PATHS = "$(TARGET_BUILD_DIR)/usr/local/lib/include" "$(OBJROOT)/UninstalledProducts/include" "$(OBJROOT)/UninstalledProducts/$(PLATFORM_NAME)/include" "$(BUILT_PRODUCTS_DIR)"
    IPHONEOS_DEPLOYMENT_TARGET = 8.0
    OTHER_LDFLAGS = -ObjC

    SWIFT_OBJC_BRIDGING_HEADER = $(PROJECT_DIR)/$(PROJECT_NAME)/Bridging-Header.h
    TARGETED_DEVICE_FAMILY = 1,2


=== BUILD TARGET CordovaLib OF PROJECT CordovaLib WITH CONFIGURATION Debug ===




Check dependencies



=== BUILD TARGET MyApp OF PROJECT MyApp WITH CONFIGURATION Debug ===


Check dependencies

warning: no rule to process file '/Users/emcniece/Code/ionic/app/platforms/ios/MyApp/Plugins/cordova-plugin-is-debug/IsDebug.h' of type sourcecode.c.h for architecture i386

“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.


** BUILD FAILED **



The following build commands failed:
  Check dependencies
(1 failure)

Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/emcniece/Code/ionic/app/platforms/ios/cordova/build-debug.xcconfig,-project,MyApp.xcodeproj,-target,MyApp,-configuration,Debug,-sdk,iphonesimulator,-destination,platform=iOS Simulator,build,CONFIGURATION_BUILD_DIR=/Users/emcniece/Code/ionic/app/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/emcniece/Code/ionic/app/platforms/ios/build/sharedpch
```

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)
 

**Which Ionic Version?** 2.x

`ionic info`:

Cordova CLI: 6.3.1
Gulp version:  CLI version 3.9.1
Gulp local:
Ionic Framework Version: 2.0.0-rc.0
Ionic CLI Version: 2.1.0
Ionic App Lib Version: 2.1.0-beta.1
ios-deploy version: 1.9.0
ios-sim version: 5.0.8
OS: Mac OS X El Capitan
Node Version: v6.7.0
Xcode version: Xcode 8.0 Build version 8A218a

Any way to get around this?

**Comment by jgw96** [comment]
This issue was moved to driftyco/ionic-native#620

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! Since this is an issue with Ionic native i have moved this issue to that repo. Thanks!

**Comment by emcniece** [comment]
Hey @jgw96 - this has been closed in driftyco/ionic-native#620, would you consider reopening here? Is there a better place to post tickets for Ionic CLI issues?
